pennylane_lightning/core/src/simulators/lightning_gpu/utils/MPI_helpers.hpp

```diff
@@ -0,0 +1,163 @@
+// Copyright 2022-2023 Xanadu Quantum Technologies Inc.
```

Comment:
Let's not do it now, but this file should be a `.cpp` file, not a `.hpp` file, as there are no templates, and everything is a standalone function. We get away with this working from the one-def-rule since everything is marked as `inline`. Can you create a backlog story to update this?

Reply:
Done!